uber/models.py

```diff
@@ -591,7 +591,7 @@ def change_badge(self, attendee, badge_type, badge_num=None):
 
                 # determine the new badge number now that the badges have shifted
                 next = self.next_badge_num(badge_type, old_badge_num)
-                badge_num = min(badge_num or next, next)
+                badge_num = min(int(badge_num) or next, next)
```

Comment:
Ah, I guess our unit tests aren't testing what happens when we pass in strings, or else they would have caught this.